src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -469,13 +471,14 @@ def freeze(self):
 
     def unfreeze(self):
         """
-        If this method is called, the frame returns into its previous interactable state.
+        If this method is called, the frame returns into its previous intractable state.
         """
         for button in self.buttons:
             button.configure(state=tkinter.NORMAL)
 
         self._frozen = False
 
+    @staticmethod
```

Comment:
why static here? and if you really want it, then remove the self

Reply:
ok